e2e/common/accessCodeHelper.ts

```diff
@@ -0,0 +1,65 @@
+import { APIRequestContext, request } from "@playwright/test";
+import { getAccessToken, getS2SToken } from "./getAccessTokenHelper";
+
+interface CaseInvite {
+  id: string;
+  value: CaseInviteDetails;
+}
+
+interface CaseInviteDetails {
+  accessCode: string;
+  isApplicant: string;
+}
+
+export class AccessCodeHelper {
+  public static async getApplicantAccessCode(ccdRef: string): Promise<string> {
+    const caseInvites: CaseInvite[] = await this.getAccessCode(ccdRef);
+    return (
+      caseInvites.find((invitee) => {
+        return invitee.value.isApplicant === "Yes";
+      })?.value.accessCode || ""
+    );
+  }
+
+  public static async getRespondentAccessCode(ccdRef: string): Promise<string> {
+    const caseInvites: CaseInvite[] = await this.getAccessCode(ccdRef);
+    return (
+      caseInvites.find((invitee) => {
+        return invitee.value.isApplicant === "No";
+      })?.value.accessCode || ""
+    );
+  }
+
+  private static async getAccessCode(ccdRef: string): Promise<CaseInvite[]> {
+    let caseInvites: CaseInvite[] = [];
+    const apiContextSystemUser: APIRequestContext = await request.newContext();
+    const apiContextS2SToken: APIRequestContext = await request.newContext();
+    const tokenSystemUserCreateCase = await getAccessToken(
+      "accessCode",
+      apiContextSystemUser,
+    );
+    const s2sToken = await getS2SToken(apiContextS2SToken, "prl_cos_api");
```

Comment:
Should this be a secret or if not, should it be declared separately?

Reply:
Are you thinking the "prl_cos_api" should be a secret?

Reply:
Not sure if it's sensitive or not, guessing not as it doesn't look to be a secret already in the service
But maybe it should be a class property

Reply:
Yeah I don't think it needs to be a secret looking at other repos. I've just moved it to a variable lmk if you're happy